pkg/utils/utils.go

```diff
@@ -122,7 +123,13 @@ func GetClusterNameFromKubectlCmd(cmd string) string {
 	if len(matchedArray) >= 2 {
 		s = matchedArray[1]
 	}
-	return s
+
+	str, err := strconv.Unquote(s)
+	if err != nil {
```

Comment:
Had to think about this, works.

Reply:
if you think I should put a comment or something, or you have better alternative, let me know 👍

Reply:
Nope all good.